backend/src/main/java/de/qaware/mercury/storage/shop/impl/ShopDataRepository.java

```diff
@@ -10,23 +10,60 @@
 public interface ShopDataRepository extends JpaRepository<ShopEntity, UUID> {
     List<ShopEntity> findByName(String name);
 
+    /**
+     * Finds all approved and enabled shops
+     *
+     * @return all approved and enabled shops
+     */
     @Query(
-        "SELECT s as shopEntity, " +
-            "sqrt(" +
-            "  power(111.3 * cos((:latitude + s.latitude) / (2 * 0.01745)) * (:longitude - s.longitude) , 2)" +
-            " + power(111.3 * (:latitude - s.latitude), 2)" +
-            ") as distance FROM ShopEntity AS s " +
-            "WHERE s.enabled = true AND s.approved = true"
+        "SELECT s as shopEntity  FROM ShopEntity AS s WHERE s.enabled = true AND s.approved = true"
```

Comment:
Einfacher wäre: `SELECT s FROM ShopEntity s WHERE s.enabled = true AND s.approved = true`

Reply:
Jo, den Teil der Query hab ich einfach nicht angepasst, der ist noch wie er war. Ich pass das an.